app/src/main/res/drawable/edit_text_background_shadow.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+  <item>
+    <shape>
+      <corners android:radius="5dp"></corners>
```

Comment:
The dimen should be in multiple of 4 and sometimes 2 is fine. Change this to `5dp`

Reply:
This is not related to the view dimension, this attribute was for the layer of shadow customisation